warehouse/query-core/src/main/java/datawave/query/jexl/functions/TLDFiAggregator.java

```diff
@@ -0,0 +1,96 @@
+package datawave.query.jexl.functions;
+
+import datawave.query.Constants;
+import datawave.query.attributes.Attribute;
+import datawave.query.attributes.AttributeFactory;
+import datawave.query.attributes.Document;
+import datawave.query.data.parsers.FieldIndexKey;
+import datawave.query.data.parsers.KeyParser;
+import datawave.query.jexl.JexlASTHelper;
+import org.apache.accumulo.core.data.Key;
+import org.apache.accumulo.core.data.Range;
+import org.apache.accumulo.core.data.Value;
+import org.apache.accumulo.core.iterators.SortedKeyValueIterator;
+import org.apache.hadoop.io.Text;
+
+import java.io.IOException;
+
+/**
+ * A field index aggregator for TLD queries
+ */
+public class TLDFiAggregator extends FiAggregator {
+
+    public TLDFiAggregator(){
+        //  empty constructor
+    }
+
+    //  apply(SortedKeyValueIterator<Key,Value> itr)
+    //  this path is handled by overriding sameUid()
+
+    //  apply(SortedKeyValueIterator<Key,Value> iter, Range range, Collection<ByteSequence> cfs, boolean includeColumnFamilies)
+    //  this path is handled by overriding getSeekRange()
```

Comment:
delete?

Reply:
Deleted in next commit, comments added to the `sameUid()` and `getSeekRange()` methods